RecoEgamma/EgammaIsolationAlgos/python/interestingEgammaIsoDetIdsSequence_cff.py

```diff
@@ -45,6 +45,26 @@
 interestingGedGamIsoDetIdEE.innerRadius = 0.0
 
 import RecoEgamma.EgammaIsolationAlgos.interestingGamIsoDetIdModule_cff
+interestingOotGamIsoDetIdEB = RecoEgamma.EgammaIsolationAlgos.interestingGamIsoDetIdModule_cff.interestingGamIsoDetId.clone()
```

Comment:
Okay, I can certainly make this change -- was simply following the format of the rest of the fragment.  Easy enough.